pdns/recursordist/docs/changelog/4.5.rst

```diff
@@ -0,0 +1,201 @@
+Changelogs for 4.5.X
+====================
+
+.. changelog::
+  :version: 4.5.0-alpha1
+  :released: 15th of January 2021
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 9699
+    :tickets: 440
+
+    Introduce "Refresh almost expired" a mechanism to keep the record cache warm.
+
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 9917
+
+    Set the Protocol Buffer 'inBytes' field for zero-length messages.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 9843
+    :tickets: 9780, 9781
+
+    Use protozero for Protocol Buffer operations in dnsdist, and dnstap/outgoing for the recursor.
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 9883
+    :tickets: 9621
+
+    Lookup DS entries before CNAME entries.
+
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 9841
+
+    ProtobufLogger: Better handling of short-reads, errors.
```

Comment:
I don't think that change is included in the recursor?

Reply:
you're right